libr/include/r_io.h

```diff
@@ -184,11 +191,29 @@ typedef struct r_io_map_t {
 	int fd;
 	int perm;
 	ut32 id;
+	ut64 ts;
```

Comment:
@RHL120 thinks timestamps should be signed, so you can define dates before 1970

Reply:
so maybe better to have our time type. what about typedef st64 tt64?

Reply:
well, the timestamps are for runtime verification of maps only

Reply:
Why not just use a counter and not depend on the rtc? Ots enough to keep them sorted by time

Reply:
i would like to not depend on time for this to work

Reply:
ping ^